### ISSUE TITLE
Sync error

### ISSUE THREAD

**Issue by yuuki0xff** [cpbotha/nvpy#158]
The login API returns 404 error.   I contacted the simplenote support team to solve this problem.

```
$ date
Tue Aug 14 21:32:06 JST 2018
$ curl -v 'https://simple-note.appspot.com/api/login?email=test@example.com&password=dummy'
*   Trying 172.217.26.20...
(omitted)
* Connection state changed (MAX_CONCURRENT_STREAMS == 100)!
< HTTP/2 404
< content-type: text/html; charset=UTF-8
< x-cloud-trace-context: xxxxxxx
< date: Tue, 14 Aug 2018 12:29:36 GMT
< server: Google Frontend
< content-length: 154
< alt-svc: quic=":443"; ma=2592000; v="44,43,39,35"
<
<html>
 <head>
  <title>404 Not Found</title>
 </head>
 <body>
  <h1>404 Not Found</h1>
  The resource could not be found.<br /><br />



 </body>
* Connection #0 to host simple-note.appspot.com left intact
</html>
```

**Comment by clach04** [comment]
@yuuki0xff any response from the support team? I did wonder if this was related to the CAPTCHA in place on the login form, hence the 404/401. However nvpy is now syncing as of today (and the curl command above now returns a 405, which seems reasonable) so I think they resolved it.

I did read at https://simplenote.com/developers/ that Legacy SimpleNotes old API will go away on 2019-10-01 replaced by Simperium.
